### ISSUE TITLE
Use Any()

### ISSUE THREAD

**Issue by marek1972** [ralmsdeveloper/EntityFrameworkCore.FirebirdSQL#21]
### The issue

Use of Any()

### Steps to reproduce

if (context.Students.Any()){}

### Error details

InnerException = {Dynamic SQL Error
SQL error code = -206
Column unknown
TRUE
At line 5, column 15}

```
Exception message:
Stack trace:
SELECT CASE
    WHEN EXISTS (
        SELECT 1
        FROM "Student" AS "s")
    THEN TRUE ELSE FALSE
END FROM RDB$DATABASE

FirebirdSql.Data.FirebirdClient.FbException (0x80004005): Dynamic SQL Error
SQL error code = -206
Column unknown
TRUE
At line 5, column 15 ---> Dynamic SQL Error
```

### Further technical details

Firebird version: 2.5
EntityFrameworkCore.FirebirdSql version: 2.0.11.3

Other details about my project setup:

**Comment by ralmsdeveloper** [comment]
Actually this was not implemented for version 2.5, only 3.x, for the reason that boolean type support.
I was planning to implement this for 2.1, because we will support the conversion, but I will try to implement this soon for FB 2.5

**Comment by marek1972** [comment]
Thanks.

**Comment by ralmsdeveloper** [comment]
Fixed in version 2.0.11.4, available in nuget!

**Comment by marek1972** [comment]
Any() works properly. Thanks a lot.
